functions/locate_files

```diff
@@ -326,6 +326,10 @@ locate_actin_database_sql_script() {
     echo "/data/resources/crunch/actin/sql_database/generate_database.sql"
 }
 
+locate_actin_database_views_sql_script() {
+    echo "/data/resources/crunch/actin/sql_database/generate_views.sql"
+}
+
```

Comment:
Might be nice to eventually tie the SQL files to actual releases. Currently we may have different VMs running different versions of ACTIN with potentially different datamodels. In scripts we only support a single global SQL though (which was fine with one version and one VM...)

I'd be perfectly fine with leaving this as it is right now btw, changes to the SQL datamodel are quite rare and our setup is not so complex that we can't coordinate such updates!

Reply:
A quick and easy improvement would actually be to extract these from the JAR. It doesn't solve all the problem (you have to remember to blow away the database on a new release with DB changes), but at least removes one duplication of the files.

In the longer run VMs + scripts are not how we'll automate DB loading tasks. Then we'll need to consider patching or a schema-per-patient approach as you've suggested earlier.